packages/perseus/src/widgets/__tests__/radio/choice-icon_test.jsx

```diff
@@ -34,41 +30,33 @@ function renderChoiceIcon(options) {
 }
 
 describe("choice icon", () => {
-    describe.each([["sat"], ["library"]])("product: %s", (product) => {
-        describe.each([[true], [false]])(
-            "multipleSelect: %s",
-            (multipleSelect) => {
-                it("renders with the correct border radius", () => {
-                    // Arrange
-                    let expectedRadius;
-                    if (multipleSelect) {
-                        expectedRadius = 3;
-                    } else if (product === "sat") {
-                        expectedRadius = SAT_ICON_SIZE;
-                    } else {
-                        expectedRadius = LIBRARY_ICON_SIZE;
-                    }
-
-                    // Act
-                    renderChoiceIcon({product, multipleSelect});
-
-                    const choiceWrapper = screen.getByTestId(
-                        `choice-icon__${product}-choice-icon`,
-                    );
-
-                    // Assert
-                    expect(choiceWrapper).toHaveStyle(
-                        `border-radius: ${expectedRadius}px`,
-                    );
-                });
-            },
-        );
+    describe.each([[true], [false]])("multipleSelect: %s", (multipleSelect) => {
+        it("renders with the correct border radius", () => {
+            // Arrange
+            let expectedRadius;
+            if (multipleSelect) {
+                expectedRadius = 3;
+            } else {
+                expectedRadius = LIBRARY_ICON_SIZE;
```

Comment:
Suggestion: Should we change this variable and similar variables to just be ICON_SIZE and remove mention of LIBRARY since the purpose of that was to differentiate between SAT and LIBRARY? I think Matthew mentions this about another section too. Also relevant for the Id in line 47

Reply:
Yes!  Matthew had the same suggestion above and I'll make this change in a new PR. :)  Thanks!